Gemfile

```diff
@@ -35,6 +35,7 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  gem 'rubocop', '~> 0.68.1', require: false
```

Comment:
PER FECT!
Remember to ALWAYS tag the version of gem that you are using. [More information here](https://guides.rubygems.org/patterns/#pessimistic-version-constraint)